great-idea/css/index.css

```diff
@@ -63,4 +63,100 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Your code starts here! */
+
+/* Index */
+
+h1 {
+    font-size: 60px;
+    text-align: center;
+}
+
+a {
+    text-decoration: none; 
+    color: black;
+    margin: 30px;
+}
+
+nav {
+    padding: 25px;
+}
+
+.pagetwo {
+    width: 100%;
+    padding: 15px;
+}
+
+.logo {
+    
+}
+
+.container {
```

Comment:
You are reusing this class in three different sections. So all three of those sections are getting the same styles when they each need to be styled differently based on the requirements of the design file.